src/components/pages/Chat/RoomNameLoader.tsx

```diff
@@ -20,13 +21,22 @@ export default function RoomNameLoader({ roomId }: Props) {
             }
 
             const stream = await streamrClient.getStream(roomId)
-
-            dispatch({
-                type: ActionType.RenameRoom,
-                payload: {
-                    [roomId]: stream.description || '',
-                },
-            })
+            try {
+                const description = getRoomDescription(stream)
```

Comment:
I'm not sure if we should call it description, getting description, etc. It's not that. It's metadata. How 'bout we call this one `getRoomMetadata` instead?